notification/src/handler/notification/notification.handler.js

```diff
@@ -160,6 +161,15 @@ function _obfuscateNotificationInfoFromActionFields(updateActions) {
   return copyOfUpdateActions
 }
 
+function getPaymentMethodName(notificationRequestItem) {
+  const paymentMethodFromNotification = notificationRequestItem.paymentMethod
+  const paymentMethodsToNames = getPaymentMethodsToNamesConfig()
+  return (
+    paymentMethodsToNames[paymentMethodFromNotification] ||
+    paymentMethodFromNotification
```

Comment:
try to escape `paymentMethodFromNotification` here as well

Reply:
I don't think we need to escape here. According to the docs there's no restrictions: https://docs.commercetools.com/api/projects/payments#paymentmethodinfo